### ISSUE TITLE
Changing current working directory while running as a module prevents loading own profiles

### ISSUE THREAD

**Issue by schmidtandreas** [archlinux/archinstall#612]
Hello!
I tried to execute archinstall as a module from iso image with own configuration, own script (similar to guided) and own profiles.
However, the archinstall did not find my profiles, although they were under `./profiles`. After short research I found the reason for that behavior. The `archinstall` changes the current working directory, if it is executed as module:

https://github.com/archlinux/archinstall/blob/3a4076419418dcdac538c543903cfc6e16bde35f/archinstall/__init__.py#L99

Since `archinstall` is located under `/usr/lib/python3.9/site-packages/archinstall`, `./profiles` becomes `/usr/lib/python3.9/site-packages/archinstall/profiles` . Apart from the fact that it is redundant, since this directory is also searched for profiles, my profiles are not found.

Of course, I can copy my profiles to `~/.config/archinstall/profiles`. In my usecase, however, it would mean that I clone the git repo and then have to copy my profiles to `~/.config/archinstall/profiles`. That is little bit ugly, IMO. It would be nicer, if I could use the profiles stored under `./profiles` in the repo, without any changes/movements.

So I have some questions:
- Why `archinstall` changes the cwd?
- How can I use my profiles in `./profiles`?

**Comment by Torxed** [comment]
> So I have some questions:
> 
> * Why `archinstall` changes the cwd?
> * How can I use my profiles in `./profiles`?

1) We [didn't always used to](https://github.com/archlinux/archinstall/blob/84410113c20f7614fcaa442723618b14352bdc4f/archinstall/__main__.py), but at some point we did to support [setup.cfg](https://github.com/archlinux/archinstall/commit/ecf7a2a237a04882c93fd4d743b037f7af6bc253) using entry points.
2) In the past, I've also placed profiles in `./profiles` so ideally I want this to work too. A workaround would be to place your profile on a webserver somewhere if possible, and point your profile to `https://wherever.com/yourprofile.py` and archinstall can source the profile from there. This is not ideal, but would get you going for now.

Apologies in the late response, I've been working my way backwards from the issues I haven't had time to catch up on.
